core/main/src/broker/endpoint_broker.rs

```diff
@@ -686,6 +686,10 @@ async fn forward_extn_event(
 }
 
 fn apply_response(result: Value, filter: String, rpc_request: &RpcRequest, v: &mut BrokerOutput) {
+    if result.is_null() {
```

Comment:
apply_response is called by passing only the `result` field of `JsonRpcApiResponse`.   A `None`  value for this field indicates an `error` response from the RPC call.  

Where are we processing the error response?   Looks like this change will mask `v.data.error = Some(r);`